FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -463,7 +476,9 @@ static bool prvCollectDeviceMetrics( void )
 {
     bool xStatus = false;
     eMetricsCollectorStatus eMetricsCollectorStatus;
```

Comment:
Variables with the same name as the type can be confusing – let's rename this variable, e.g.:

```c
eMetricsCollectorStatus eStatus;
```